test/e2e/scale.go

```diff
@@ -83,11 +87,19 @@ type scaleSpecInput struct {
 	// Can be overridden by variable CAPI_SCALE_CONTROLPLANE_MACHINE_COUNT.
 	ControlPlaneMachineCount *int64
 
-	// WorkerMachineCount defines number of worker machines to be added to each workload cluster.
+	// WorkerMachineCount defines number of worker machines per machine deployment of the workload cluster.
 	// If not specified, 1 will be used.
 	// Can be overridden by variable CAPI_SCALE_WORKER_MACHINE_COUNT.
 	WorkerMachineCount *int64
 
+	// MachineDeploymentCount defines the number of MachineDeployments to be used per workload cluster.
+	// If not specified, 1 will be used.
+	// Can be overridden by variable CAPI_SCALE_MACHINE_DEPLOYMENT_COUNT.
+	// Note: This assumes that the cluster template of the specified flavor has at least one machine deployment.
+	// It uses this machine deployment to create additional copies.
+	// Names of the MachineDeployments will be overriden to "md-1", "md-2", etc.
```

Comment:
```suggestion
	// Names of the MachineDeployments will be overridden to "md-1", "md-2", etc.
```
Didn't know golangci-lint does spellchecking as well :D